pyiron_base/jobs/job/runfunction.py

```diff
@@ -456,6 +454,19 @@ def run_job_with_runmode_srun(job):
     )
 
 
+def run_static_decorator(func):
```

Comment:
The name is not very clear, maybe `set_runtime` or something like that.

Reply:
I renamed the decorator to `run_time_decorator`.